src/Utilities/Compiler/Analyzer.Utilities.projitems

```diff
@@ -32,7 +32,7 @@
     <Compile Include="$(MSBuildThisFileDirectory)CodeMetrics\CodeAnalysisMetricData.PropertyMetricData.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)CodeMetrics\ComputationalComplexityMetrics.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)CodeMetrics\MetricsHelper.cs" />
-    <Compile Include="$(MSBuildThisFileDirectory)CodeMetrics\SemanticModelProvider.cs" />
```

Comment:
Is this file deleted in this PR or is it still used somewhere?

Reply:
It's been renamed as `CodeMetricsAnalysisContext`. It was used only for code metrics so it seemed ok to do a rename instead of adding a new class.